### PR TITLE
Programming languages: add docs for javascript and other

### PR DESCRIPTION
**Pull request by NolanRoose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,47 @@
...
+  + __Editor tools__
+    - [Sublime snippets](https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#sublime-text)
+- [Ionic](https://ionicframework.com/)
+- [Angular 2](https://angular.io/docs/ts/latest/api/)
+  + __Best practises__
+    - [John papa Style Guide](https://github.com/johnpapa/angular-styleguide/blob/master/a2/README.md)
+- [Ionic 2](http://ionicframework.com/docs/v2/api/)
+- [GulpJS](http://gulpjs.com/)
+
+## Programming Languages
```
Comment (flexbox) [inline README.md:45]: Can you add the link on the Table-of-content ?

**Review by flexbox — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -21,7 +24,46 @@
...
+    - [D3](https://d3js.org/)
+    - [Lodash](https://lodash.com/)
+    - [JSHint](http://jshint.com/)
+    - [DropZoneJS](http://www.dropzonejs.com/)
+    - [Toastr](https://github.com/CodeSeven/toastr)
+    - [DataTables](https://www.datatables.net/)
+  + __Editor tools__
+    - [Sublime snippets](https://github.com/zenorocha/sublime-javascript-snippets)
+
+## Plateforms
```
Comment (mghignet) [inline README.md]: Platforms* :stuck_out_tongue:

File: README.md [hunk 3]
```diff
@@ -6,7 +6,10 @@
 
 - [Design Resources](#design-resources)
 - [Icons](#icons)
-- [Front-end](#front-end)
+- [Front-End Development](#front-end-development)
+- [Programming Languages](#programming-languages)
+- [Plateforms](#plateforms)
```
Comment (mghignet) [inline README.md]: Platforms*

**Comment by flexbox** [comment]
🚀

**Pull request merged**